middle_end/flambda2/terms/code_metadata.mli

```diff
@@ -33,9 +33,9 @@ module type Code_metadata_accessors_result_type = sig
 
   val params_arity : 'a t -> Flambda_arity.t
 
-  val num_leading_heap_params : 'a t -> int
+  (* val num_leading_heap_params : 'a t -> int *)
```

Comment:
Let's remove this if it isn't needed

Reply:
Ah yes, thanks. I thought to cleanup the `.ml` but forgot the `.mli`.